Ambiguous-Squids/bangazon-cli
# As password

## Description

Allow a created customer to login/logout of the Bangazon ordering system with password encryption functionality.

## Related Issue
See
#49
#50
#51 
#52 

## Motivation and Context
Allows encrypted passwords in database, in case database access is compromised.  Bcrypt is the library used to encrypt passwords.

## How Has This Been Tested?

- Add user, enter password
- Select user in menu system
- Enter password

View database to see if password is encrypted properly by inspecting hash

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help!
